.github/workflows/validate.yml

```diff
@@ -77,10 +77,22 @@ jobs:
       - name: Install Python Dependencies
         run: pip3 install -r espressopp/requirements.txt
 
-      - name: build documentation
+      - name: build espressopp
         run: |
           cmake -S espressopp -B espressopp-build
-          cmake --build espressopp-build --target doxygen sphinx sphinx-pdf -j 2
```

Comment:
Do we still need to split this now?

Reply:
No, but since the output is quite lengthy I find it easier to debug this way.